app/views/courses/CourseVideosComponent.vue

```diff
@@ -31,7 +39,11 @@
 
       .row.m-t-5
         .col-md-6
-          iframe.video(src="https://player.vimeo.com/video/310626807" frameborder= "0" webkitallowfullscreen mozallowfullscreen allowfullscreen)
```

Comment:
Just wanted to highlight that I really like that you're grabbing the video urls from the `utils.coffee`.
Thanks for improving the code.